pkg/config/property.go

```diff
@@ -49,4 +49,8 @@ const (
 
 	workspaceIdleTimeout        = "devworkspace.idle_timeout"
 	defaultWorkspaceIdleTimeout = "15m"
+
+	// Skip Verify For Insecure connection with TLS
+	tlsSkipVerifyInsecure        = "tls.skip_verify_insecure"
```

Comment:
Just wonder why you choose the different order than go has
`insecure_tls_verify` vs `tls_verify_insecure`. It does not make any difference but could be consistent.